test/res.sendFile.js

```diff
@@ -93,6 +93,26 @@ describe('res', function(){
       .expect('Content-Type', 'application/x-bogus')
       .end(done);
     })
+    
+    it('should set etag according to app settings', function (done) {
```

Comment:
Please add a test that checks the behavior when someone sets the etag setting to `strong`, `weak`, or a custom function.